tests/auth_role_moderation_test.py

```diff
@@ -27,7 +27,8 @@ def test_moderation(auth_fc):
     auth_fc.sr_sim_type_set("myapp")
     with srunit.quest_start() as qcall:
         qcall.auth_db.model("UserRole").delete_all()
```

Comment:
There's a pattern here. If you find yourself exposing assumptions in code, e.g. "all users must have ROLE_USER", then something is wrong. The delete_all calls (in all cases) was from "lazy" coding. Since tests always create new databases, this works, but it makes an assumption about the roles table that it shouldn't.

In this case, what's wanted is to delete the sym_type role and moderation record for myapp for this particular user.